build/config/PublishData.json

```diff
@@ -0,0 +1,36 @@
+{
+  "master": {
+    "nugetPush": [
```

Comment:
"automaticNuGetPush", since we always know we're potentially doing pushes in other forms?